Errors fixed and an updated some information (#5273)
qgis/QGIS-Documentation: docs/training_manual/vector_classification/classification.rst

   :align: center

* Click :guilabel:`OK`.

> * You can save the colour ramp by selecting :guilabel:`Save Color Ramp...` under the

Author: It'll hurt a little to train myself out of colours, greys and '-ise's but I'll keep a check on it ;)

> * You can save the colour ramp by selecting :guilabel:`Save Color Ramp...` under the

Reviewer: > It'll hurt a little to train myself out of colours, greys and '-ise's but I'll keep a check on it ;)

I am also more comfortable with British spelling. I think we should allow both US and British spelling in the documentation - the users will not have any problems with that, and it seems strange that US spelling should be preferred when most developers, community members and users are from Europe.

   :align: center

* Click :guilabel:`OK`.

> * You can save the colour ramp in by selecting :guilabel:`Save Color Ramp...` under the :guilabel:`Color ramp` tab. Choose an appropriate name for the colour ramp. You will now be able to select the same colour ramp easily under :guilabel:`All Color Ramps`.

Reviewer: ```suggestion
* You can save the colour ramp by selecting :guilabel:`Save Color Ramp...` under the
  :guilabel:`Color ramp` tab.
  Choose an appropriate name for the colour ramp.
  You will now be able to select the same colour ramp easily under
  :guilabel:`All Color Ramps`.
```
Avoid long lines (70 to 80 characters maximum)

* Click the :guilabel:`Add rule` button: |signPlus|.
* A new dialog then appears.

> * Click the epsilon :guilabel:`Ɛ` button next to the :guilabel:`Filter` text area.

Reviewer: > .. |expression| image:: /static/common/mIconExpression.png
>    :width: 1.5em

I took the liberty to add this...

.. image:: img/rule_style_result.png
   :align: center

> * Add a new criterion :kbd:`"landuse" <> 'residential' AND "AREA" >= 1000000`

Reviewer: ```suggestion
* Add a new criterion :kbd:`"landuse" <> 'residential' AND "AREA" >= 0.00005`
```
The problem here is that the CRS is EPSG:4326, so we are talking about square degrees! And 1 000 000 is an awful lot of square degrees ;-) I am not sure if 0.00005 is OK, but it can't be 1 000 000.

Author: I am not sure what's going on there. I have followed the tutorial and all my $AREA outputs are in the range 130 to 1000000000. Perhaps QGIS is automatically calculating $AREA in square metres. 

Have tried changed the 'Units for area measurement' under Project Properties to no avail. 

If I figure it out I may send another pull request with a 'Note: Be sure you're calculating area in degrees by...'.

Thanks for the help!

Reviewer 2: @gafrgwynedd what's the CRS of your project? I quickly browsed previous lessons and loaded some files and it's unclear what should be the expected CRS of the project (or the downloaded files); i see 4326, 3857 and I think somewhere there'd be a south-african crs.

  and choose a mid-green color.
* Add another new criterion :kbd:`"name"  =  'Swellendam'` and assign it
  a darker grey-blue color in order to indicate the town's importance in the
  region.
* Click and drag this criterion to the top of the list.

These filters are exclusive, in that they collectively exclude some areas on the

> map (i.e. those which are smaller that 1000000, are not residential and are not

Reviewer: ```suggestion
map (i.e. those which are smaller than 0.00005 (square degrees!), are not residential and are not
```
